contracts/deploy/mainnet/120_pool_booster_curve_arb.js

```diff
@@ -0,0 +1,137 @@
+      "CurvePoolBooster",
+      addresses.zero
+    );
+    const cCurvePoolBoosterProxy = await ethers.getContractAt(
+      "CurvePoolBoosterProxy",
+      addresses.zero
+    );
+
+    console.log("\nStarting deployment using CreateX");
+    const rewardToken = addresses.mainnet.OUSDProxy;
+    const gauge = addresses.mainnet.CurveOUSDUSDTGauge;
+    const targetedChainId = 42161; // arbitrum
+    const fee = 0;
+
+    // Get CreateX contract
+    const cCreateX = await ethers.getContractAt(createxAbi, addresses.createX);
+
+    // Generate encoded salt (deployer address || crossChainProtectionFlag || bytes11(keccak256(rewardToken, gauge)))
+    const addressDeployerBytes20 = ethers.utils.hexlify(
+      ethers.utils.zeroPad(deployerAddr, 20)
+    );
+    const crossChainProtectioFlagBytes1 = ethers.utils.hexlify(
+      ethers.utils.zeroPad(0, 1)
+    );
+    const saltBytes11 =
+      "0x" +
+      ethers.utils
+        .keccak256(
+          ethers.utils.concat([
+            ethers.utils.arrayify(rewardToken),
+            ethers.utils.arrayify(gauge),
+          ])
+        )
+        .slice(2, 24);
+    const encodedSalt = ethers.utils.hexlify(
+      ethers.utils.concat([
+        addressDeployerBytes20,
+        crossChainProtectioFlagBytes1,
+        saltBytes11,
+      ])
+    );
+    console.log(`Encoded salt: ${encodedSalt}`);
+
+    // --- Deploy implementation --- //
+    const cachedInitCodeImpl = ethers.utils.concat([
+      PoolBoosterBytecode.bytecode,
+      ethers.utils.defaultAbiCoder.encode(
+        ["uint256", "address", "address"],
+        [targetedChainId, rewardToken, gauge]
+      ),
+    ]);
+    const txResponse = await withConfirmation(
+      cCreateX.connect(sDeployer).deployCreate2(encodedSalt, cachedInitCodeImpl)
+    );
+    const txReceipt = await txResponse.wait();
+    // event 0 is GovernorshipTransferred
+    // event 1 is ContractCreation, topics[1] is the address of the deployed contract, topics[2] is the salt
+    const implementationAddress = ethers.utils.getAddress(
+      `0x${txReceipt.events[1].topics[1].slice(26)}`
+    );
+    console.log(
+      `Curve Booster Implementation deployed at: ${implementationAddress}`
+    );
+
+    // --- Deploy and init proxy --- //
+    const cachedInitCodeProxy = ProxyBytecode.bytecode; // No constructor arguments
+    const initializeImplem = cCurvePoolBooster.interface.encodeFunctionData(
+      "initialize(address,uint16,address,address)",
+      [
+        addresses.base.multichainStrategist, // strategist
+        fee, // fee
+        addresses.base.multichainStrategist, // feeCollector
+        addresses.mainnet.CampaignRemoteManager, // campaignRemoteManager
+      ]
+    );
+    const initializeProxy = cCurvePoolBoosterProxy.interface.encodeFunctionData(
+      "initialize(address,address,bytes)",
+      [
+        implementationAddress, // implementation
+        addresses.mainnet.Timelock, // governor
```

Comment:
we don't have a timelock on Arbitrum yet right?

Reply:
Indeed, I just copy/pasted the code from mainnet to be sure that this doesn't break cross-chain replication.
However we can setup another owner of the proxy in Arbitrum, it will not change deployment address.

And, yes, we don't have `Timelock` on arbitrum. Any suggestion for which address should I use instead?

Reply:
5/8 multisig on Arbitrum would be a good one: https://docs.originprotocol.com/registry/contracts#arbitrum